main/FatesConstantsMod.F90

```diff
@@ -47,8 +47,24 @@ module FatesConstantsMod
   ! with the host model.
   integer, public, parameter :: prescribed_n_uptake = 1
   integer, public, parameter :: coupled_n_uptake    = 2
-
   integer, public, parameter :: coupled_np_comp_scaling = 1 ! This flag signals that at least 1 chemical element (ie N or P)
+  
+  !Flags specifying how tree regeneration works
+  
+  integer, public, parameter :: TRS_no_seedling_dyn = 3                          ! Constant defining the Tree Recruitment
+                                                                                 ! Scheme switch. This value turns on 
+                                                                                 ! size-based reproductive allocation 
+                                                                                 ! and allocation to non-seed 
+                                                                                 ! reproductive biomass, but does not turn 
+                                                                                 ! on seedling dynamics.
+  integer, public, parameter :: TRS = 2                                          ! Constant defining the Tree Recruitment
```

Comment:
recommend calling this trs_regeneration, to be consistent with "default_regeneration" and "regeneration_model"

Reply:
Ok, agreed. I have changed this.